addons/web_editor/static/src/scss/web_editor.backend.scss

```diff
@@ -59,7 +59,7 @@
         z-index: 1001 !important;
         border: 0;
     }
-    > :not(.modal):not(.modal-backdrop) {
+    .tooltip-field-info {
```

Comment:
I can't be sure 100% but from the manual tests I've done, it seems to be good.
I would rather have a blacklist of element to hide rather than a whitelist of element to show because adding anything that floats in the body would usually be intended to be shown on top of the mass_mailing fullscreen (eg. toolbars, powerbox, odoo commandbar, media modal, ...).

Reply:
I we see in the future an element that is not hidden, we would add it as a blacklisted element.